test_data/mysql_shard_bulk.cfg

```diff
@@ -0,0 +1,21 @@
+{
+    "configType":"bulk",
+    "shardConfigurationBulk":{
+       "schemaShard":{
+          "host":"test",
+          "user":"test",
+          "password":"test",
+          "port":"3306",
+          "dbName":"test"
+       },
+       "otherShards":[
```

Comment:
Lets keep the naming inline with the dataflow config

Reply:
Done